NeuralNetwork/tick_tack_toe.cpp

```diff
@@ -10,7 +10,7 @@ void flip_vec(std::vector<float>& game_vec)
 }
 
 //takes highest number returned from network, and translates into an int 0-8
-int interpret(std::vector<float>& decision_vec)
+int interpret(const std::vector<float>& decision_vec)
```

Comment:
On `tick_tack_toe.cpp:141`, you call `interpret` with a temporary value (rvalue), which cannot be referenced by a non-const reference (yikes, rules are gross). But turns out `interpret` only needs a const reference, which can be bound to an rvalue. See [here](http://en.cppreference.com/w/cpp/language/value_category) for more info.